docs/reference/tags/create_line_item.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: default
+title: Create Line Item
+parent: Tags
+has_children: false
+---
+
+The Create Line Item Tag renders a form which acts as a wrapper for a single item.
+
+##### input
+{% raw %}
+```liquid
+    {% form "create_line_item" %}
+        <input type="hidden" data-variant-id="{{item_id}}" name="items[][variant_id]" value="{{item_id}}"/>
```

Comment:
I don't think the `data-variant-id` is a requirement for this form

Reply:
I'll update this 🙌